test/e2e/storage/external/external.go

```diff
@@ -255,6 +255,10 @@ func (d *driverDefinition) SkipUnsupportedTest(pattern testpatterns.TestPattern)
 		if d.SnapshotClass.FromName || d.SnapshotClass.FromFile != "" || d.SnapshotClass.FromExistingClassName != "" {
```

Comment:
Why is the empty string case on L252 mean that it's supported?

Reply:
git blame shows this commit https://github.com/kubernetes/kubernetes/commit/6644db9914379a4a7b3d3487b41b2010f226e4dc#diff-e5fc8d9911130b421b74b1ebc273f458R206

which is the first commit for this file. I don't know, perhaps @pohly can help.

Reply:
It might have been needed for some tests which don't set a type explicitly, but I don't remember exactly. Most of the documentation for `SnapshotType` is just of the variety "XXXType is a type for XXX", which doesn't really help: https://github.com/kubernetes/kubernetes/blob/6644db9914379a4a7b3d3487b41b2010f226e4dc/test/e2e/storage/testpatterns/testpattern.go#L48-L63